<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QToasterTunerLCD</class>
 <widget class="QWidget" name="QToasterTunerLCD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>115</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="titleWidget" native="true">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>25</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#titleWidget
{
	background-color: rgb(50, 50, 50);
}</string>
   </property>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>50</width>
      <height>15</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>167</red>
          <green>179</green>
          <blue>175</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>167</red>
          <green>179</green>
          <blue>175</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>167</red>
          <green>179</green>
          <blue>175</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>167</red>
          <green>179</green>
          <blue>175</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>167</red>
          <green>179</green>
          <blue>175</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>167</red>
          <green>179</green>
          <blue>175</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>167</red>
          <green>179</green>
          <blue>175</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>167</red>
          <green>179</green>
          <blue>175</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>167</red>
          <green>179</green>
          <blue>175</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#titleLabel
{
	color: rgb(167, 179, 175);
}</string>
    </property>
    <property name="text">
     <string>TUNER</string>
    </property>
   </widget>
   <widget class="QLabel" name="noteLabel">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>20</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#noteLabel
{
	color: rgb(167, 179, 175);
}</string>
    </property>
    <property name="text">
     <string>G#</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="octaveLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#octaveLabel
{
	color: rgb(167, 179, 175);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="masterTuneTitle">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>94</y>
     <width>95</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Master Tune</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="masterTuneVal">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>82</y>
     <width>95</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;0.0&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="scaleOuterWidget" native="true">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>25</y>
     <width>286</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#scaleOuterWidget
{
	background-color: rgb(50, 50, 50);
}</string>
   </property>
   <widget class="QLabel" name="indexLabel">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>10</y>
      <width>254</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label	
{
	background-color: rgb(167, 179, 175);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line1">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>10</y>
      <width>2</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#line1
{
	color:  rgb(50, 50, 50);
	background-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line2">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>10</y>
      <width>2</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#line2
{
	color:  rgb(50, 50, 50);
	background-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
